### PR TITLE
Migrate to wdio assertions library

### PR DESCRIPTION
**Pull request by VirajP1002**
### What is the context of this PR?
The purpose of this PR is to remove the usages of chai and use wdio's builtin assertion library to make assertions work as intended

### How to review
- Run functional tests to see if passes
- Check the changes to ensure webdriver syntax is used and is correct
- Potentially converting a single file with the tool but this may be difficult as manual intervention is required and I added my own improvements which isn't done by the tool so the results will differ

### Checklist
* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by VirajP1002** [comment]
[Using wdio's builtin assertion library instead of chai (2 SP)](https://trello.com/c/jENNSOdY/5846-using-wdios-builtin-assertion-library-instead-of-chai-2-sp)

File: tests/functional/spec/features/placeholder/playback_confirmation.spec.js [hunk 1]
```diff
@@ -10,8 +10,6 @@ describe("Feature: Playback Confirmation", () => {
     await $(MandatoryCheckboxPage.ham()).click();
     await click(MandatoryCheckboxPage.submit());
 
-    await expect(await $("#confirm-answers-question ul").getHTML())
-      .to.contain("Cheese")
-      .to.contain("Ham");
+    await expect(await $("#confirm-answers-question ul").getHTML()).toContain("Ham");
```
Comment (katie-gardner) [inline tests/functional/spec/features/placeholder/playback_confirmation.spec.js:13]: Looks like the tool didn't handle a chained replacement here, do you remember having to deal with this one manually? if not might be worth searching for any other occurences

File: tests/functional/spec/thank_you.spec.js [hunk 2]
```diff
@@ -49,24 +49,22 @@ describe("Thank You Default View Response Enabled", () => {
...
     });
 
     it("When I navigate to the thank you page, and I have submitted more than 40 seconds ago, Then I shouldn't see the option to view my answers", async () => {
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.false;
-      await browser.pause(46000); // Waiting 40 seconds for the timeout to expire (45 minute timeout changed to 35 seconds by overriding VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS for the purpose of the functional test)
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.true;
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).getHTML()).to.contain("For security, you can no longer view or get a copy of your answers");
+      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).toBe(false);
+      await browser.pause(50000); // Waiting 40 seconds for the timeout to expire (45 minute timeout changed to 35 seconds by overriding VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS for the purpose of the functional test)
```
Comment (katie-gardner) [inline tests/functional/spec/thank_you.spec.js:65]: why did the brower pause change?

File: tests/functional/spec/features/placeholder/playback_confirmation.spec.js (hunk 1 above)
Comment (VirajP1002) [reply tests/functional/spec/features/placeholder/playback_confirmation.spec.js:13]: I don't remember having to do it manually, I think the tool removed it. Yeah, I can have look at that

File: tests/functional/spec/thank_you.spec.js (hunk 2 above)
Comment (VirajP1002) [reply tests/functional/spec/thank_you.spec.js:65]: That was Pete and I testing something, I completely forgot about this, will revert

File: tests/functional/spec/timeout/timeout_modal.js [hunk 3]
```diff
@@ -6,28 +6,20 @@ class TestCase {
...
-      await expect(await browser.getUrl()).to.contain("/session-expired");
-      await expect(await $("body").getHTML())
-        .to.include(
-          "Sorry, you need to sign in again",
-          "This is because you have either:",
-          "been inactive for 45 minutes and your session has timed out to protect your information",
-          "followed a link to a page you are not signed in to",
-          "followed a link to a survey that has already been submitted",
-        )
-        .to.not.include("To protect your information, your progress will be saved and you will be signed out in");
```
[the remark this answers is not in the document]
Comment (VirajP1002) [reply tests/functional/spec/timeout/timeout_modal.js:18]: Ahh okay, I missed this one also, I can add another statement, hopefully that should work

**Review by katie-gardner — APPROVED** [review]
Haven't found any other anomalies and my comments resolved 👍 Looking good

**Review by liamtoozer — APPROVED** [review]
Looks great from looking through most of the files, and passes for me - nice work! 

It might be out of scope for this ticket, as I'm not sure what the impact might be, but would it be worth removing the chai [dev dependency](https://github.com/ONSdigital/eq-questionnaire-runner/blob/main/package.json#L35) and [chai linter](https://github.com/ONSdigital/eq-questionnaire-runner/blob/main/package.json#L39) as part of this work too?

**Review by berroar — APPROVED** [review]

**Comment by MebinAbraham** [comment]
> Looks great from looking through most of the files, and passes for me - nice work!
> 
> It might be out of scope for this ticket, as I'm not sure what the impact might be, but would it be worth removing the chai [dev dependency](https://github.com/ONSdigital/eq-questionnaire-runner/blob/main/package.json?rgh-link-date=2023-10-30T13%3A45%3A51Z#L35) and [chai linter](https://github.com/ONSdigital/eq-questionnaire-runner/blob/main/package.json?rgh-link-date=2023-10-30T13%3A45%3A51Z#L39) as part of this work too?

Yes, those should now be removed.

**Pull request merged**